app/build.gradle

```diff
@@ -69,6 +69,9 @@ android {
         targetCompatibility 1.8
         sourceCompatibility 1.8
     }
+    configurations.all {
+        resolutionStrategy.cacheChangingModulesFor 0, 'seconds'
+    }
```

Comment:
Do we want this in production? I know we've needed it lately due to library stuff on our side but it probably isn't a long-term thing we want.